### ISSUE TITLE
.po files

### ISSUE THREAD

**Issue by hugok79** [BodhiDev/bodhi_translations#13]
Should po files not be updated according to recent commits? 
* 3f65e06fdd77de7b72138b438fb0339b3bfd6657
* 568cab02e00b1c6577c0daff229be99d3730c2f5

**Comment by rbtylee** [comment]
Yes probably. And it all needs to be brought up to date with my new quickstart_local repo. But this project seems unmaintained and I am too busy at the moment to deal with it. I will leave this open until someone can address these issues. Or I find time to address them myself.

**Comment by hugok79** [comment]
@rbtylee shouldn't it be replaced by this one https://github.com/rbtylee/quickstart_local

**Comment by hugok79** [comment]
Perhaps this https://github.com/BodhiDev/bodhi_translations repository should be listed as a `Public Archive`or deleted...

Since we have https://github.com/BodhiDev/bodhi7packages/tree/main/bodhi/bodhi-quickstart
